flask_project/campaign_manager/insights_functions/_abstract_overpass_insight_function.py

```diff
@@ -56,32 +56,21 @@ def get_data_from_provider(self):
         :rtype: dict
         """
         features = self.feature.split('=')
+        coordinates = self.campaign.get_task_boundary_as_geoJSON()
```

Comment:
those 3 lines are being repeated as well in _abstract_overpass_user_function.py
we could create a function (that we can test) to avoid DRY principle (Don't Repeat Yourself) and to avoid bugs.